scheduling/allocator.rb

```diff
@@ -130,7 +130,7 @@ def self.update_vm(vm_host, vm)
         allocated_at: Time.now
       )
       AssignedVmAddress.create_with_id(dst_vm_id: vm.id, ip: ip4.to_s, address_id: address.id) if ip4
-      vm.sshable&.update(host: vm.ephemeral_net4 || vm.ephemeral_net6.nth(2))
+      vm.sshable&.update(host: vm.ephemeral_net4 || NetAddr.parse_net(vm.ephemeral_net6).nth(2))
```

Comment:
The problem is... heroku doesn't support IPv6. So, this sshable wouldn't work in prod